host-interaction/file-system/windows-file-protection/bypass-windows-file-protection.yml

```diff
@@ -6,7 +6,7 @@ rule:
       - michael.hunhoff@mandiant.com
     scopes:
       static: function
-      dynamic: span of calls
+      dynamic: thread
```

Comment:
this is a dup of #987, which I'd prefer to merge from there. would you please revert this change?